cloud/deployment/deployment.go

```diff
@@ -210,6 +216,13 @@ func Create(label, workspaceID, description, clusterID, runtimeVersion string, s
 		return errors.Wrap(err, astro.AstronomerConnectionErrMsg)
 	}
 
+	if waitForStatus {
+		err = healthPoll(d.ID, workspaceID, client)
+		if err != nil {
+			return err
```

Comment:
nitpick: should we just log the error and still display the information about the deployment which is being created? 
Even if there was a timeout checking for the deployment to come up, would it be easier from the user's point of view if we still show the deployment info and provide the link for the user to get to the deployment?